Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/CalibrationAPI.cs

```diff
@@ -0,0 +1,389 @@
+            }
+            catch
+            {
+                PrintLastError();
+            }
+
+            return false;
+        }
+
+        public static List<MarkerCorners> CalcMarkerCornersRelativeToCamera(HeadsetCalibrationData data)
+        {
+            List<MarkerCorners> markersRelativeToCamera = new List<MarkerCorners>();
+            var cameraTransform = Matrix4x4.TRS(data.headsetData.position, data.headsetData.rotation, Vector3.one);
+            var inverseCameraTransform = cameraTransform.inverse;
+            foreach (var markerPair in data.markers)
+            {
+                var arUcoCorners = markerPair.arucoMarkerCorners;
+
+                var cornersRelativeToCamera = new MarkerCorners();
+                cornersRelativeToCamera.topLeft = inverseCameraTransform.MultiplyPoint(arUcoCorners.topLeft);
+                cornersRelativeToCamera.topRight = inverseCameraTransform.MultiplyPoint(arUcoCorners.topRight);
+                cornersRelativeToCamera.bottomRight = inverseCameraTransform.MultiplyPoint(arUcoCorners.bottomRight);
+                cornersRelativeToCamera.bottomLeft = inverseCameraTransform.MultiplyPoint(arUcoCorners.bottomLeft);
+                var orientationTransform = inverseCameraTransform * Matrix4x4.TRS(Vector3.zero, arUcoCorners.orientation, Vector3.one);
+                cornersRelativeToCamera.orientation = Quaternion.LookRotation(orientationTransform.GetColumn(2), orientationTransform.GetColumn(1));
+                markersRelativeToCamera.Add(cornersRelativeToCamera);
+            }
+            return markersRelativeToCamera;
+        }
+
+        public List<CalculatedCameraExtrinsics> CalculateIndividualArUcoExtrinsics(CameraIntrinsics intrinsics, int numExtrinsics)
+        {
+            int intrinsicsSize = 12;
+            float[] inputIntrinsics = new float[intrinsicsSize];
+            inputIntrinsics[0] = intrinsics.FocalLength.x;
+            inputIntrinsics[1] = intrinsics.FocalLength.y;
+            inputIntrinsics[2] = intrinsics.PrincipalPoint.x;
+            inputIntrinsics[3] = intrinsics.PrincipalPoint.y;
+            inputIntrinsics[4] = intrinsics.RadialDistortion.x;
+            inputIntrinsics[5] = intrinsics.RadialDistortion.y;
+            inputIntrinsics[6] = intrinsics.RadialDistortion.z;
+            inputIntrinsics[7] = intrinsics.TangentialDistortion.x;
+            inputIntrinsics[8] = intrinsics.TangentialDistortion.y;
+            inputIntrinsics[9] = intrinsics.ImageWidth;
+            inputIntrinsics[10] = intrinsics.ImageHeight;
+            inputIntrinsics[11] = 0.0f; // reprojection error, unused
+
+            int sizeExtrinsics = 7;
+            float[] extrinsics = new float[numExtrinsics * sizeExtrinsics];
+            if (!ProcessIndividualArUcoExtrinsicsNative(inputIntrinsics, extrinsics, numExtrinsics))
+            {
+                PrintLastError();
+                return null;
+            }
+            else
+            {
```

Comment:
Nit: This else isn't strictly necessary, and is also not consistent with how things are done in the rest of this file.

i.e. CalculateGlobalArUcoExtrinsics has something similar and does without the else.

While I think I'd prefer the model that CalculateGlobalArUcoExtrinsics has, I think it's more important to be consistent here so it's up to you which model you pick.